packages/discovery-provider/src/api/v1/users.py

```diff
@@ -219,6 +228,56 @@ def get(self, id):
         return get_single_user(user_id, current_user_id)
 
 
+tracks_route_parser = current_user_parser.copy()
+tracks_route_parser.add_argument(
+    "id", action="append", required=False, description="The ID of the user(s)"
+)
+
+
+@ns.route("")
+class BulkUsers(Resource):
+    @record_metrics
+    @ns.doc(
+        id="""Get Bulk Users""",
+        description="Gets a list of users by ID",
+        responses={200: "Success", 400: "Bad request", 500: "Server error"},
+    )
+    @ns.expect(tracks_route_parser)
+    @ns.marshal_with(users_response)
+    @cache(ttl_sec=5)
+    def get(self):
+        args = tracks_route_parser.parse_args()
+        ids = decode_ids_array(args.get("id"))
+        users = get_users({"id": ids})
+        if not users:
```

Comment:
would we want the same thing for bulk playlists?

Reply:
Ah yes probably!